src/types/workbook_data.ts

```diff
@@ -74,7 +74,7 @@ export interface WorkbookData {
   sheets: SheetData[];
   styles: { [key: number]: Style };
   formats: { [key: number]: Format };
-  borders: { [key: number]: Border };
+  borders: { [key: number]: NewBorderData };
```

Comment:
Double check at the end of the dev if it does not require a migration (for now, no)